### PR TITLE
Create decred_memes

### PR DESCRIPTION
**Pull request by OfficialCryptos**

### CODE REVIEW

File: decred_memes [hunk 1]
```diff
@@ -0,0 +1,12 @@
+| Title               | decred meme collection |
+| Author              | OfficialCryptos |
+| Featured text       |Decred meme|
+
+Thesis statement for art collection - What the article is about, and what is it aiming to prove
+
+# Article Title
+
+Meme Piece 1
```
Comment (OfficialCryptos) [inline decred_memes:9]: meme interstellar dcr

File: decred_memes [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Meme Piece 2
```
Comment (OfficialCryptos) [inline decred_memes:10]: DCR Guy

Comment (PhoenixGreen) [reply decred_memes:10]: Love the image but we’re not going to be able to use this one due to copyright issues.

File: decred_memes (hunk 1 above)
Comment (PhoenixGreen) [reply decred_memes:9]: Is this image from a movie? Just a heads up, we can’t publish images that are subject to copyright E.g. anything from TV shows, movies, games etc.

**Pull request merged**